src/test/validation_tests.cpp

```diff
@@ -49,13 +50,13 @@ BOOST_AUTO_TEST_CASE(subsidy_limit_test)
 {
     const auto chainParams = CreateChainParams(CBaseChainParams::MAIN);
     CAmount nSum = 0;
-    for (int nHeight = 0; nHeight < 14000000; nHeight += 1000) {
+    for (int nHeight = 0; nHeight < 3350000; nHeight += 500) {
```

Comment:
The case was that with halving at 52500 blocks, increment by 1000 results in overflow and wrong calculation of more than 21M coins minted